src/data/fintype/basic.lean

```diff
@@ -231,6 +231,9 @@ if hα : nonempty α then by letI := classical.inhabited_of_nonempty hα;
   exact of_surjective (inv_fun f) (inv_fun_surjective H)
 else ⟨∅, λ x, (hα ⟨x⟩).elim⟩
 
+noncomputable instance of_subtype [fintype α] (p : α → Prop) : fintype (subtype p) :=
```

Comment:
Since `fintype (subtype p)` is the conclusion instead of the hypothesis:
```suggestion
noncomputable instance subtype [fintype α] (p : α → Prop) : fintype (subtype p) :=
```

Reply:
(oops, I missed that I already suggested the change! fixed now.)